website/content/docs/connect/intentions/jwt-authorization.mdx

```diff
@@ -12,6 +12,8 @@ By specifying a JSON Web Key Set (JWKS) in the configuration entry and referenci
 
 ## Workflow
 
+~> **Note**: Consul must be configured to use ACLs with a default deny policy for JWT authorization with service intentions to work.
```

Comment:
```suggestion
```
Same. However, we can add this information as a step 1 in the process starting at line 19. Something like:

"1. Verify that ACLs are enabled and that the [`default_policy`]() is set to `deny`."

It would still feel a little odd to put this level of detail here, but if people are missing this detail then we need to expose it at a higher level.